scripts/process/process-hydrograph.R

```diff
@@ -61,39 +60,46 @@ process.plot_hydrographTotal <- function(viz=getContentInfo("NMHydrograhTotal-sv
     rects <- xml_find_all(rect_svg_all, '//*[local-name()="rect"]')
     rect_svg <- rects[length(rects)]
     
-    xml_set_attr(rect_svg, 'id', paste0("b",yr))
+    xml_set_attr(rect_svg, 'data-year', as.character(yr))
```

Comment:
Looks like it is supported for newer browsers: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/data-*. Sorry Laura, if we want to support IE we will need to go back to using the id. Just make sure the ids are unique.